cli.js

```diff
@@ -37,21 +37,31 @@ const printAllStyles = () => {
 };
 
 const cli = meow(`
-	Usage
-	  $ chalk <style> … <string>
-	  $ echo <string> | chalk --stdin <style> …
+	${chalk.greenBright.inverse(' Usage ')}
 
-	Options
-	  --template, -t    Style template. The \`~\` character negates the style.
-	  --stdin           Read input from stdin rather than from arguments.
-	  --no-newline, -n  Don't emit a newline (\`\\n\`) after the input.
-	  --demo            Demo of all Chalk styles.
+	  $ ${chalk.green('chalk')} ${chalk.yellow('[options…]')} ${chalk.cyan('<style> … <string>')}
+	  $ echo ${chalk.cyan('<string>')} | ${chalk.green('chalk')} ${chalk.yellow('--stdin [options…]')} ${chalk.cyan('<style> …')}
```

Comment:
For consistency, I think `echo` should be green too.

Reply:
Ah interesting. So:

- green = command
- yellow = options (flags)
- cyan = arguments

?

Reply:
Yes

Reply:
Done in 5dcdf2a.